app/addons/documents/rev-browser/rev-browser.components.react.jsx

```diff
@@ -0,0 +1,446 @@
+        isDiffViewEnabled: store.getIsDiffViewEnabled(),
+        databaseName: store.getDatabaseName()
+      };
+    }
+
+    componentDidMount () {
+      store.on('change', this.onChange, this);
+    }
+
+    componentWillUnmount () {
+      store.off('change', this.onChange);
+    }
+
+    onChange () {
+      this.setState(this.getStoreState());
+    }
+
+    toggleDiffView (enableDiff) {
+      RevActions.toggleDiffView(enableDiff);
+    }
+
+    render () {
+      const {tree, ours, theirs, dropdownData, conflictingRevs, isDiffViewEnabled} = this.state;
+
+      if (!tree) {
+        return null;
+      }
+
+      // no conflicts happened for this doc
+      if (!theirs || !conflictingRevs.length) {
+        return <div style={{textAlign: 'center', color: '#fff'}}><h2>No conflicts</h2></div>;
+      }
+
+      return (
+        <div className="revision-wrapper scrollable">
+          <RevisionBrowserControls {...this.state} />
+          <div className="revision-view-controls">
+            <ButtonGroup className="two-sides-toggle-button">
+              <Button
+                style={{width: '120px'}}
+                className={isDiffViewEnabled ? 'active' : ''}
+                onClick={this.toggleDiffView.bind(this, true)}
+              >
+                <i className="icon-columns" /> Diff
+              </Button>
+              <Button
+                style={{width: '120px'}}
+                className={isDiffViewEnabled ? '' : 'active'}
+                onClick={this.toggleDiffView.bind(this, false)}
+              >
+                <i className="icon-file-text" /> Document
+              </Button>
+            </ButtonGroup>
+          </div>
+
+          {isDiffViewEnabled ?
+            <RevisionDiffArea ours={ours} theirs={theirs} /> :
+            <SplitScreenArea ours={ours} theirs={theirs} /> }
+        </div>
+      );
+    }
+  };
+
+
+  class SplitScreenArea extends React.Component {
+
+    constructor (props) {
+      super(props);
+    }
+
+    componentDidUpdate () {
+      this.hightlightAfterRender();
+    }
+
+    componentDidMount () {
+      this.hightlightAfterRender();
+    }
+
+    hightlightAfterRender () {
+      const format = (input) => { return JSON.stringify(input, null, '  '); };
+
+      const jsmode = new JavaScriptMode();
+      const left = ReactDOM.findDOMNode(this.refs.revLeftOurs);
+      const right = ReactDOM.findDOMNode(this.refs.revRightTheirs);
+
+      const leftRes = highlight.render(format(this.props.ours), jsmode, theme, 0, true);
+      left.innerHTML = leftRes.html;
+      const rightRes = highlight.render(format(this.props.theirs), jsmode, theme, 0, true);
+      right.innerHTML = rightRes.html;
+    }
+
+    render () {
+      const {ours, theirs} = this.props;
+
+      if (!ours || !theirs) {
+        return <div></div>;
+      }
+
+      return (
+        <div className="revision-split-area" style={{padding: '20px 15px', display: 'flex', color: '#fff'}}>
```

Comment:
May be better to move this to less or you'll need to include all the cross-browser rules for flex here (see `.display-flex()` in assets/less/mixins.less).